modules/gltf/src/lib/api/gltf-scenegraph.ts

```diff
@@ -50,12 +50,13 @@ export class GLTFScenegraph {
   byteLength: number;
 
   // TODO - why is this not GLTFWithBuffers - what happens to images?
-  constructor(gltf?: {json: GLTF; buffers?: any[]}) {
+  constructor(gltf?: {json: GLTF; buffers?: any[]; images?: any[]}) {
```

Comment:
```suggestion
  constructor(gltf?: GLTFWithBuffers) {
```
See comment above

Reply:
Declined. The problem is that GLTFWithBuffers has REQUIRED field "buffers", which results in necessity to change many calls of the constructor, where "gltf" doesn't have "buffers" assigned.